tutorial/README.md

```diff
@@ -13,31 +13,41 @@ These labs provide an insight on the advantages of using Helm over using Kuberne
   * Revision management
   * Repositories and chart sharing
 
+# Helm Status
```

Comment:
Since it's same as main README, can we just provide a link something like "Refer to the README Helm Status section" so that we don't have any inconsistency in future accidentally?

Reply:
Good idea, updated.